libethereum/ClientBase.cpp

```diff
@@ -417,9 +417,11 @@ BlockHeader ClientBase::pendingInfo() const
 
 BlockDetails ClientBase::pendingDetails() const
 {
-    auto pm = postSeal().info();
-    auto li = Interface::blockDetails(LatestBlock);
-    return BlockDetails((unsigned)pm.number(), li.totalDifficulty + pm.difficulty(), pm.parentHash(), h256s{});
+    auto pendingHeader = postSeal().info();
+    auto latestDetails = Interface::blockDetails(LatestBlock);
+    return BlockDetails{(unsigned)pendingHeader.number(),
+        latestDetails.totalDifficulty + pendingHeader.difficulty(), pendingHeader.parentHash(),
+        h256s{} /* children */, static_cast<unsigned>(postSeal().blockData().size())};
```

Comment:
(`static_cast` will not be needed if you change the type of `blockSizeBytes` to `size_t`)

Reply:
@gumb0: had originally used size_t but the returned block sized were zero for some reason..will try again and root cause why.

Reply:
Looks like that was a different issue that I've since addressed, size_t works

Reply:
Actually looks like that was an issue that I've since addressed, it works now

Reply:
So can `static_cast` me removed?